src/blockdata/opcodes.rs

```diff
@@ -652,48 +652,99 @@ impl fmt::Debug for All {
+                        (all::OP_NOP1.code <= self.code &&
+                        self.code <= all::OP_NOP10.code) {
+                    Class::NoOp
+                // 87 opcodes
+                } else if self.code == 80 || self.code == 98 ||
+                        (self.code >= 126 && self.code <= 129) ||
+                        (self.code >= 131 && self.code <= 134) ||
+                        (self.code >= 137 && self.code <= 138) ||
+                        (self.code >= 141 && self.code <= 142) ||
+                        (self.code >= 149 && self.code <= 153) ||
+                        (self.code >= 187 && self.code <= 254) {
+                    Class::SuccessOp
+                // 1 opcode
+                } else if self == all::OP_RETURN{
+                    Class::ReturnOp
+                // 1 opcode
+                } else if self == all::OP_PUSHNUM_NEG1 {
+                    Class::PushNum(-1)
+                // 16 opcodes
+                } else if all::OP_PUSHNUM_1.code <= self.code &&
+                        self.code <= all::OP_PUSHNUM_16.code {
+                    Class::PushNum(1 + self.code as i32 - all::OP_PUSHNUM_1.code as i32)
+                // 76 opcodes
+                } else if self.code <= all::OP_PUSHBYTES_75.code {
+                    Class::PushBytes(self.code as u32)
+                // 61 opcodes
+                } else {
+                    Class::Ordinary(Ordinary::try_from_all(self).unwrap())
+                },
+            ClassifyContext::Legacy =>
+                if self == all::OP_VERIF || self == all::OP_VERNOTIF ||
+                    self == all::OP_CAT || self == all::OP_SUBSTR ||
+                    self == all::OP_LEFT || self == all::OP_RIGHT ||
+                    self == all::OP_INVERT || self == all::OP_AND ||
+                    self == all::OP_OR || self == all::OP_XOR ||
+                    self == all::OP_2MUL || self == all::OP_2DIV ||
+                    self == all::OP_MUL || self == all::OP_DIV || self == all::OP_MOD ||
+                    self == all::OP_LSHIFT || self == all::OP_RSHIFT {
+                    Class::IllegalOp
+                // 11 opcodes
+                } else if self == all::OP_NOP ||
+                        (all::OP_NOP1.code <= self.code &&
+                            self.code <= all::OP_NOP10.code) {
+                    Class::NoOp
+                // 75 opcodes
+                } else if self == all::OP_RESERVED || self == all::OP_VER || self == all::OP_RETURN ||
+                        self == all::OP_RESERVED1 || self == all::OP_RESERVED2 ||
+                        self.code >= all::OP_CHECKSIGADD.code {
+                    Class::ReturnOp
+                // 1 opcode
+                } else if self == all::OP_PUSHNUM_NEG1 {
+                    Class::PushNum(-1)
+                // 16 opcodes
+                } else if all::OP_PUSHNUM_1.code <= self.code &&
+                        self.code <= all::OP_PUSHNUM_16.code {
```

Comment:
Nit:

`all::PUSHNUM.code..=all::OP_PUSHNUM_16.code.contains(self.code)` is more idiomatic; otherwise we will have clippy warning here (when we will start using it)

Reply:
Will change it. It was a copy-paste of the currently existing code

Reply:
`.contains` does not work for 1.29 :(

Reply:
Added to #510 so we don't forget it afterwards.